### DISCUSSION TITLE
Mac menu bar icon

### DISCUSSION THREAD

**Discussion by rkscodes** [cpeditor/cpeditor#1103] [Q&A]
Is there any way to disable the cpeditor icon in  menu bar from mac ?
<img width="1437" alt="Screenshot 2022-09-19 at 11 27 00 AM" src="https://user-images.githubusercontent.com/30290728/190957146-42c87a16-a142-497b-89dc-c26d3dfae801.png">

**Comment by ghost** [comment]
Cpeditor does not seem to have this setting, but you can use software like Bartender4 to hide it
